draft-ietf-tls-tls13.md

```diff
@@ -1901,7 +1902,7 @@ which signature algorithms may be used in digital signatures. Clients which
 desire the server to authenticate itself via a certificate MUST send this extension.
 If a server
 is authenticating via a certificate and the client has not sent a
-"signature_algorithms" extension then the server MUST
+"signature_algorithms" extension then, the server MUST
 abort the handshake with a "missing_extension" alert
```

Comment:
This comma, though, I disagree with.  (Maybe it was intended to be before "then" and not after?)